cmd/kops/create.go

```diff
@@ -28,8 +30,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/kops/cmd/kops/util"
 	kopsapi "k8s.io/kops/pkg/apis/kops"
+	"k8s.io/kops/pkg/apis/kops/registry"
 	"k8s.io/kops/pkg/apis/kops/v1alpha1"
+	"k8s.io/kops/pkg/apis/kops/validation"
+	"k8s.io/kops/upup/pkg/fi"
```

Comment:
I couldn't find a reference to this new import in the code: `"k8s.io/kops/upup/pkg/fi"`; does this import import a single object/class or is it more like a namespace? The other new imports all occured in the code (e.g. "validate") so this stood out to me.